google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/resources/GoogleUserModelItem.java

```diff
@@ -203,6 +206,8 @@ private void loadUserProjects() {
               GoogleUserModelItem.this.add(item);
             }
 
+            // Re-applies the filter to the new child nodes.
+            setFilter(filter);
```

Comment:
should we check for null here? I know that ResourceProjectModelItem takes a nullable filter, but maybe this could same some cycles?

Reply:
Done.